runtime/mainnet/src/filters.rs

```diff
@@ -32,11 +32,6 @@ impl Contains<RuntimeCall> for MainnetCallFilter {
 			return false;
 		}
 
-		match call {
-			// Block jobs and roles pallet
-			RuntimeCall::Roles(_) | RuntimeCall::Jobs(_) => false,
```

Comment:
Do we want to block services pallet initially?

Reply:
Yes